### PR TITLE
docs: added pull request template

### PR DESCRIPTION
**Pull request by erinmcj**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] The changes have been reviewed for accessibility compliance.
+ 
+#### Testing
+- [ ] New functionality is tested
+- [ ] All unit and integration tests passed
+- [ ] The changes have been reviewed for accessibility compliance.
+ 
+#### Deployment
+- [ ] Documentation has been updated where needed
+- [ ] Any required database migrations have been added and tested
```
Comment (swalker125) [inline pull_request_template.md]: no db migrations in this repo

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] I have performed a self-review of my code.
+- [ ] Code changes are considered complete for the given story (i.e., the comments from code review are addressed)
+- [ ] The commit title follows conventional commit standards.
+- [ ] I have put the PR link into the planner.
+- [ ] The PR has a clear and descriptive title and a detailed description of the changes.
+- [ ] Code builds cleanly without any errors or warnings
+ 
+#### Code Quality
+- [ ] I have run linting and there were no errors
+- [ ] The changes have been reviewed for accessibility compliance.
```
Comment (swalker125) [inline pull_request_template.md]: what does this mean?

**Review by swalker125 — COMMENTED** [review]
Few comments

File: pull_request_template.md (hunk 2 above)
Comment (erinmcj) [reply pull_request_template.md]: I was trying to say the changes to the code pass accessibility checks. I will reword this so it makes more sense!

File: pull_request_template.md (hunk 1 above)
Comment (erinmcj) [reply pull_request_template.md]: oops missed this when trying to make it the same as the backend. I'll fix that

**Review by thomkainos — APPROVED** [review]

**Review by swalker125 — APPROVED** [review]

**Pull request merged**